conf/config.yaml

```diff
@@ -1,4 +1,4 @@
 defaults:
-  - data: cartpole_st1_at.yaml
-  - model: SVR.yaml
-  - simulator: gboost_cartpole.yaml
+  - data: moab_100epi.yaml
```

Comment:
you haven't commited `moab_100epi.yaml`. Do note that you can also keep the defaults in `conf/config.yaml` and then use the CLI to pass new configs, e.g.: `python ddm_trainer.py data=moab_100epi`

Reply:
I am removing the sim folder, and adding an entry point to `ddm_test_validate`.  Based on our discussion earlier today, we can separate the ddm_predictor into ddm_connector for bonsai platform integration and use `ddm_test_validate` to do random policy comparison with simulator at exactly same frequency. For this PR, I am just making the changes to ddm_test_validate: 1) removing sim,  2) adding entry point for user to include their sim locally, and   3) adding logging config to all simulator configs, and rest of comments below.

Reply:
Updated these changes with recent commit.